src/control/server/README.md

```diff
@@ -0,0 +1,351 @@
+
+Parameters passed to `daos_server` on the commandline as application options (excluding environment variables) take precedence over values specified in config file.
+
+For convenience, active parsed config values are written to the directory where the server config file was read from or `/tmp/` if that fails.
+
+If user shell executing `daos_server` has environment variable `CRT_PHY_ADDR_STR` set, user os environment will be used when spawning `daos_io_server` instances. In this situation an error message beginning "using os env vars..." will be printed and no environment variables will be added as specified in the `env_vars` list within the per-server section of the server config file. This behaviour provides backward compatibility with historic mechanism of specifying all parameters through environment variables.
+
+It is strongly recommended to specify all parameters and environment for running DAOS servers in the [server config file](https://github.com/daos-stack/daos/tree/master/utils/config/daos_server.yml).
+
+To clarify with respect to environment variables affecting the behaviour of `daos_io_server` instances:
+
+* If the trigger environment variable is set in the user's shell, the control plane will use the environment variables set in the shell.
+The config file will be ignored.
+
+* If the trigger environment variable is NOT set in the user's shell, the shell environment variables will be overridden by the parameters set in the config file.
+
+### Configuring logging through the config file
+
+Log file and level mask for both data (`daos_io_server`) and control (`daos_server`) planes can be set in the server config file.
+TODO: examples for global (control) and per-server (data) log parameters in config including DAOS specific syntax (D_LOG_MASK=ERR,mgmt=DEBUG).
+
+### Selecting and formatting NVMe/block device storage through the config file
+
+Parameters prefixed with `nvme_` in the per-server section of the config file determine how NVMe storage will be assigned for use by DAOS on the storage node.
+TODO: examples for NVMe and AIO (emulation) classes including config file syntax and results/logging of formatting operations including generated nvme.conf file and SPDK device format through language bindings.
+
+### Selecting and formatting SCM/pmem storage through the config file
+
+Parameters prefixed with `scm_` in the per-server section of the config file determine how SCM storage will be assigned for use by DAOS on the storage node.
+TODO: examples for both DCPM and RAM (emulation) SCM classes including config file syntax and results/logging of formatting operations.
+
+## Subcommands
+
+`daos_server` supports various subcommands (see `daos_server --help` for available subcommands) which will perform stand-alone tasks as opposed to launching as a daemon (default operation if launched without subcommand).
+
+### prep-nvme
+
+This subcommand requires elevated permissions and needs to be run with root permissions (sudo).
```

Comment:
Feels a little redundant - maybe cut down to "This subcommand requires elevated permissions (sudo)."